metadata.json

```diff
@@ -21,15 +21,13 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "5",
```

Comment:
Is this the right PR to drop these?

Reply:
cherry-picked into https://github.com/puppetlabs/puppetlabs-apache/pull/1824